### PR TITLE
Update huggingface token classification example

### PR DESCRIPTION
**Pull request by strickvl**
Updated the example to use some default values. (The example wasn't actually broken if you use the command listed in the `README.md`, but it fails when you try running the pipeline without specifying all the default values.) It now includes some core defaults (as per the documented suggestions) so it should run without errors for anyone not referring to the documentation.

Uncommenting the integration test will have to wait until other changes are made elsewhere.

## Pre-requisites
Please ensure you have done the following:
- [ ] I have read the **CONTRIBUTING.md** document.
- [ ] If my change requires a change to docs, I have updated the documentation accordingly.
- [ ] If I have added an integration, I have updated the [integrations](https://docs.zenml.io/component-gallery/integrations) table and the [corresponding website section](https://zenml.io/integrations).
- [ ] I have added tests to cover my changes.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Other (add details above)

### CODE REVIEW

File: examples/huggingface/run.py [hunk 1]
```diff
@@ -103,13 +103,15 @@ def main(
...
         token_classification_config = HuggingfaceParameters(
             label_all_tokens=True,
-            pretrained_model=pretrained_model,
-            epochs=epochs,
-            batch_size=batch_size,
+            pretrained_model=pretrained_model or "distilbert-base-uncased",
+            epochs=epochs or 1,
+            batch_size=batch_size or 16,
             dummy_run=not full_set,
+            dataset_name=kwargs.get("dataset_name") or "conll2003",
```
Comment (schustmi) [inline examples/huggingface/run.py:112]: I just checked the code and all of these already contain defaults from the click option? Wouldn't this `or` never be called?

Same for text_column and label_column

File: examples/huggingface/run.py [hunk 2]
```diff
@@ -103,13 +103,15 @@ def main(
...
 ):
     if nlp_task == "token-classification":
-        # Run Pipeline
         token_classification_config = HuggingfaceParameters(
             label_all_tokens=True,
-            pretrained_model=pretrained_model,
-            epochs=epochs,
-            batch_size=batch_size,
+            pretrained_model=pretrained_model or "distilbert-base-uncased",
+            epochs=epochs or 1,
```
Comment (schustmi) [inline examples/huggingface/run.py:109]: `epochs` is an int and will therefore never be optional, the `or` won't be called (unless someone passes 0 but that's just a user error that should crash IMO)

File: examples/huggingface/run.py [hunk 3]
```diff
@@ -103,13 +103,15 @@ def main(
...
+            batch_size=batch_size or 16,
```
Comment (schustmi) [inline examples/huggingface/run.py:110]: Same as epochs I think

File: examples/huggingface/run.py [hunk 4]
```diff
@@ -103,13 +103,15 @@ def main(
     **kwargs
 ):
     if nlp_task == "token-classification":
-        # Run Pipeline
         token_classification_config = HuggingfaceParameters(
             label_all_tokens=True,
-            pretrained_model=pretrained_model,
-            epochs=epochs,
-            batch_size=batch_size,
+            pretrained_model=pretrained_model or "distilbert-base-uncased",
```
Comment (schustmi) [inline examples/huggingface/run.py:108]: Similar to the others, this already has a default value.

**Review by schustmi — CHANGES_REQUESTED** [review]
All of my comments relate to the same issue: These values already get default values by the click options defined on the function. Maybe some of those (especially the ones which need to be different for token vs sequence classification, e.g. `dataset_name`) should not have a default value and then set the correct default dynamically?

**Review by safoinme — APPROVED** [review]

File: examples/huggingface/run.py [hunk 5]
```diff
@@ -100,17 +97,33 @@ def main(
     batch_size: int,
     epochs: int,
     full_set: bool,
-    **kwargs
+    max_seq_length: int,
+    init_lr: float,
+    weight_decay_rate: float,
+    text_column: str,
+    label_column: str,
+    dataset_name: str,
```
[the remark this answers is not in the document]
Comment (strickvl) [reply examples/huggingface/run.py]: Yep you're right.

**Review by schustmi — APPROVED** [review]

**Pull request merged**